awesomemotive/WordPress-Importer
# Make sure that the term meta value to log is a string

## Description

This PR fixes the PHP notice when import non-`string` / non-`scalar` term.

## Testing procedure

1. [Enable debug log](https://wordpress.org/documentation/article/debugging-in-wordpress/#example-wp-config-php-for-debugging) in your WordPress.
2. Add a term meta to any existing term in the site you are getting **export** from. You can use this snippet.
```php
add_term_meta(
		18, // Replace with existing term ID
		'test_non_scalar_meta',
		[
			'你好世界',
			'我是米可',
			'مرحبا بالعالم',
		]
	);
```
2. Export the content through WP Dashboard -> Tools -> Export -> All Content.
3. Perform the import.
4. Confirm that you are not getting any PHP warning in your logs after the import.
